packages/owl-bot/src/copy-code.ts

```diff
@@ -787,7 +787,7 @@ export async function copyCode(
   sourceCommitHash: string,
   destDir: string,
   workDir: string,
-  yaml: OwlBotYaml,
+  yamls: OwlBotYaml[],
```

Comment:
Any other caller other than the cli?

Reply:
Nope.  Other code paths call other functions within copy-code.ts.